### PR TITLE
[all] Make fuzz targets world executable

### PR DESCRIPTION
**Pull request by fmeum**
@jonathanmetzman In response to https://github.com/google/oss-fuzz/pull/5747#discussion_r630615419.

Making fuzz targets only user executable as the root user makes them non-executable outside the container, e.g. in the oss-fuzz/build/ dir.

### CODE REVIEW

**Comment by fmeum** [comment]
@DavidKorczynski This PR is hitting a `check_build` failure for `httplib2`'s `fuzz_request` fuzzer due to an import of the `tests` module, which does not seem to be packaged with the fuzzer in the project's `build.sh`.

**Comment by DavidKorczynski** [comment]
> @DavidKorczynski This PR is hitting a `check_build` failure for `httplib2`'s `fuzz_request` fuzzer due to an import of the `tests` module, which does not seem to be packaged with the fuzzer in the project's `build.sh`.

Thanks @fmeum - I fixed it here https://github.com/google/oss-fuzz/pull/5754

**Comment by DavidKorczynski** [comment]
@fmeum the fix has been deployed, could you retrigger the CI here with a nop commit?

**Review by DavidKorczynski — APPROVED** [review]
LGTM, I will wait for another review on this one too in case there was a specific reason to have `chmod u+x`

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**
